tests/test_main.py

```diff
@@ -98,3 +98,11 @@ def test_new_line_at_start_returns():
     res = decrypt(Config().library_default().complete_config(), "\npass\n")
 
     assert res.lower() == "\npass\n"
+
+
+def test_galactic():
```

Comment:
Can you write a test for:

> It works 100% of the time for P, Y and T and 80% of the time for X. Sometimes, for some weird reason that has to do with unicode diacritics, the dot will not be replaced. I'll have to look into that but I'm sure I can find a fix.

The 80% part? Like a separate test? It's important to test against things we fail against :) If your CI fails completely (as in, neither of us can get it to work) Please do this:

1. Make a GitHub issue about Standard Galatic failing
2. Comment out the test
3. Above the test, leave a comment with a link to the GitHub issue :)

Reply:
Did it!
https://github.com/Ciphey/Ciphey/issues/262